deepmd/dpmodel/model/linear_atomic_model.py

```diff
@@ -62,6 +62,10 @@ def get_rcut(self) -> float:
         """Get the cut-off radius."""
         return max(self.get_model_rcuts())
 
+    def get_type_map(self) -> Optional[List[str]]:
+        """Get the type map."""
+        return self.models[0].get_type_map()
```

Comment:
should be considered if the models does not have the same typemap, then how to handle. 
1. does not accept and raise an error
2. building a new typemap on top of all the models and correctly map the types to submodels?

Reply:
There is no interface for users to assign a type map...

Reply:
I would suggest raising `NotImplementedError` to indicate the model does not provide the interface at the moment.

Reply:
Done in e8ceb781b7bb77e5e7892bc87760d32b426a4180. It still needs to be implemented, otherwise `DeepEval` won't work.